Tasks/AzureFunctionAppV1/README.md

```diff
@@ -30,7 +30,7 @@ The task does not work with the Azure Classic service endpoint and it will not l
 
 ## Deployment
 
-Several deployment methods are available in this task. Web Deploy (msdeploy.exe) is the default option. To change the deployment option, expand Additional Deployment Options and enable Select deployment method to choose from additional package-based deployment options.
+Several deployment methods are available in this task. To change the deployment option, expand Additional Deployment Options and enable Select deployment method to choose from additional package-based deployment options.
```

Comment:
By default we auto-detect the best option depending on the app kind and package type provided.